README.md

```diff
@@ -387,6 +388,40 @@ E.g. [Docker Hub](https://hub.docker.com/), [Quay.io](https://quay.io/repository
    finishes successfully, and that your public repository contains the images pushed by
    the pipeline.
 
+
+#### Automatically Provision Quay Repositories For Container Images
+
+**Note**: This step is mandatory for importing components using the UI.
+
+Konflux integrates with the [Image Controller](https://github.com/konflux-ci/image-controller)
+that can automatically create Quay repositories when onboarding a component.
+
+The controller requires a token for a Quay application that has the following permissions on an
+organization:
```

Comment:
This suggests that we're going to describe the permissions, but instead we provide a procedure.

Reply:
done